docs/AboutUs.md

```diff
@@ -7,53 +7,45 @@ We are a team based in the [School of Computing, National University of Singapor
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
-## Project team
+## CoLAB Project team
 
-### John Doe
+### Fang Samuel
```

Comment:
I think putting my full name here will be better! (consistent with the other names)

`Fang Junwei, Samuel`

Reply:
Agreed. Full name for everyone would be great.

Reply:
@vevek Am i addressing your full name correctly

Reply:
@Eriksen2411 Yes :)